### PR TITLE
Feature/core 1001 sidebar design

### PR DESCRIPTION
**Pull request by Tschuck**

### CODE REVIEW

**Comment by datrinh** [comment]
Padding isnt quite right yet
image

**Comment by datrinh** [comment]
Overlays seem to be a little buggy:
image

File: core/ui.libs/src/style/dapp-wrapper.scss [hunk 1]
```diff
@@ -46,74 +54,33 @@ $sidebar-small-width: 64px;
...
+      padding: 32px 0;
       padding-bottom: 7px;
 
-      background-color: cssVar('bg-inverted');
-
       .sidebar-header {
         display: none;
       }
 
+      // only the sidepnale toggle only on small devices
```
Comment (datrinh) [inline core/ui.libs/src/style/dapp-wrapper.scss]:
```suggestion
      // sidepanel toggle only on small devices
```

File: dapps/profile.vue/src/components/root/root.ts [hunk 2]
```diff
@@ -108,8 +108,8 @@ export default class ProfileRootComponent extends mixins(EvanComponent) {
    * Load currents profiles data.
    */
   async setupProfile(): Promise<void> {
-    const { address } = this.$route.params;
     const runtime = this.getRuntime();
+    const address = this.$route.params.address || runtime.activeIdentity;
```
Comment (datrinh) [inline dapps/profile.vue/src/components/root/root.ts]: In which situations is the address missing?

Comment (Tschuck) [reply dapps/profile.vue/src/components/root/root.ts]: It's for the wallet... This one is currently excluded from the second level profile navigation and is opened via the dapp-wrapper, always in the context of the current user. Maybe we need to exclude the wallet later to a seperated DApp.

**Pull request merged**